plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts

```diff
@@ -18,6 +18,8 @@ export const entityMock = {
     namespace: 'default',
     annotations: {
       'backstage.io/managed-by-location':
+      'url:https://github.com/mcalus3/sample-service/blob/master/backstage5.yaml',

```

Comment:
That's a real username, perhaps remove it ? (I know it was there before)

Reply:
That is a good point, maybe we should consider making changes in all of the plugins containing it

Reply:
No harm but no rush or urgency on it though.